langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -215,6 +306,14 @@ public isolated function getContent(ProcessingInstruction|Comment x) returns str
 
 # Creates a new xml element item.
 #
+# ```ballerina
+# xml:createElement(
+#     "book", 
+#     {title: "Ballerina", author: "Anjana"}, 
+#     xml `<year>2022</year>`
+# ) ⇒ <book title="Ballerina" author="Anjana"><year>2022</year></book>
```

Comment:
IMO, makes more sense to have year as an attribute and author as the member.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/9dc46b7c36c28b92b9ce360ed71621b3a6eff8c0